sorc/chgres_cube.fd/surface.F90

```diff
@@ -1,24 +1,25 @@
- module surface
+!> @file
+!! @brief Process surface and nst fields.
+!!
+!! @author gayno NCEP/EMC
+!!
+!! Abstract: Process surface and nst fields.  Interpolates fields from
+!!    the input to target grids.  Adjusts soil temperature according
+!!    to differences in input and target grid terrain.  Rescales
+!!    soil moisture for soil type differences between input and target
+!!    grid.  Computes frozen portion of total soil moisture.
+!!
+!! Public Subroutines:
+!! -----------------
+!! - surface_driver -         Driver routine to process surface/nst data
```

Comment:
You can remove the list of public subroutines everywhere. Doxygen already figures that out from the code. The documentation describing each subroutine can become the first sentence of the documentation of that subroutine.

Reply:
Done.